packages/react/src/components/Storage/FileUploader/Previewer/__tests__/Previewer.test.tsx

```diff
@@ -4,9 +4,81 @@ import { render } from '@testing-library/react';
 import { Previewer } from '..';
 
 describe('Previewer', () => {
+  let fakeFile;
+  let fakeFile2;
+  beforeEach(() => {
+    fakeFile = new File(['hello'], 'hello.png', { type: 'image/png' });
+    fakeFile2 = new File(['goodbye'], 'goodbye.png', {
+      type: 'image/png',
+    });
+  });
   it('exists', async () => {
-    const { container } = render(<Previewer />);
+    const { container } = render(
+      <Previewer
+        acceptedFileTypes={['.png']}
+        files={[fakeFile]}
+        inDropZone={false}
+        multiple={true}
+        onClose={() => ''}
+        onDragEnter={() => ''}
+        onDragLeave={() => ''}
+        onDragOver={() => ''}
+        onDragStart={() => ''}
+        onDrop={() => ''}
+        onFileChange={() => ''}
```

Comment:
Should these be `() => {}` to denote empty functions? 

`() => ''` does not match `() => void` either, I'm surprised TS doesn't complain.

Reply:
I'll change it.